frontend/src/nonAuthenticated/index.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import MainRoutes from "../components/MainRoutes";
+import SideBar from "../components/SideBar";
+
+const NonAuthenticatedApp = props => {
```

Comment:
unused props - remove them
```suggestion
const NonAuthenticatedApp =() => {
```

Reply:
deleted as requested